ui/packages/statement/src/pages/StatementDetailPage.tsx

```diff
@@ -1,33 +1,37 @@
 import React from 'react'
-import { StatementDetail } from './components'
+import { StatementDetail } from '../components'
 import { useLocation } from 'react-router-dom'
-import client from '@/utils/client'
+import { DefaultApi } from '@pingcap-incubator/dashboard_client'
```

Comment:
In fact it can now, DefaultApi is just a class type here, not an instance, the Component uses its instance defined in the root project's utils/client.js:

```js
const dashboardClient = new DashboardClient.DefaultApi({
  basePath: DASHBOARD_API_URL,
  apiKey: () => authUtil.getAuthTokenAsBearer(),
})
```

Reply:
Sorry, I didn't get it, in fact it can perform unauthorized redirection now, user can add interceptors to axios when starts.